app/dashboard/ethelo.py

```diff
@@ -0,0 +1,66 @@
+from grants.models import GrantQuerySet, Grant
+
+
+EXPORT_FILENAME = "grants_export_for_ethelo.json"
+
+
+def get_grants_from_database(start_grant_number: int, end_grant_number: int=None, inactive_grants_only: bool=True) -> dict:
+    """Query the grants database and return a JSONify-able dict that can be uploaded into ethelo.
+
+    Args:
+        start_grant_number (int): The index of the starting grant.
+        end_grant_number (int, optional): The index of the ending grant. If None, all grants after `start_grant_number` 
+            will be included. Defaults to None.
+        inactive_grants_only (bool): If True, only grants that have not been activated by admins yet will be exported.
+
+    Returns:
+        dict: The returned dict will be JSONify-able.
+    """
+
+    query = GrantQuerySet(Grant)
+
+    if inactive_grants_only:
+        query.inactive()
```

Comment:
what's the actual right way to do this? on the main gitcoin.co website users have to have their grants explicitly approved, but in my ad-hoc testing this wasn't the case.

is this the proper way to check if a grant has been admin-activated?

Reply:
@kevinrolsen